wagtail_localize/wagtail_hooks.py

```diff
@@ -208,8 +208,15 @@ def is_shown(self, request, context):
                 target_locale_id=context['instance'].locale_id,
                 enabled=False
             ).exists()
+    
+    
+    class DisabledTranslationSnippetActionMenuItem():
+        def is_shown(self, request, context):
```

Comment:
Might be simpler to move the ``if issubclass(model, TranslatableMixin):`` check itself into the ``is_shown`` method on ``RestartTranslationSnippetActionMenuItem`` and then make the hook always return a ``RestartTranslationSnippetActionMenuItem``?